AdyenDropIn/DropInActionComponent.swift

```diff
@@ -100,6 +100,11 @@ public final class DropInActionComponent: ActionComponent {
     }
     
     private func perform(_ action: AwaitAction) {
+        guard environment.clientKey != nil else {
+            // swiftlint:disable:next line_length
+            assertionFailure("Failed to instantiate AwaitComponent because client key is not configured, Please supply the client key in the PaymentMethodsConfiguration if using DropInComponent, or DropInActionComponent.environment.clientKey if using DropInActionComponent separately.")
```

Comment:
`DropInActionComponent.environment.clientKey if using DropInActionComponent separately`

Maybe we can move `clientKey` as calculated property of DropInActionComponent? For convenience.
`environment.clientKey` is a bit more "internal" implementation detail..

Reply:
👍 good idea